### ISSUE TITLE
[helpme] DataLad Cmd Error: NoDatasetFound

### ISSUE THREAD

**Issue by vsoch** [datalad/datalad-helpme#5]
**What were you trying to do?**

I'm updating the reported issue to include an identifier, traceback, and command args.

**Is there anything else you want to tell us**

We would do the check for the issue existing (I think) best with a GitHub action (not really possible on the command line, esp. without a GitHub token). Also, the error is non sensical for the client because I had to artificially reproduce it (I don't know how to get to that last statement without doing this).

----------------------------------------------------------------
### Metadata
**Command :star:**
```bash
/home/vanessa/anaconda3/bin/datalad --cmd search
```
**Error Message :name_badge:**
```python
Traceback (most recent call last):
  File "/home/vanessa/Desktop/Code/datalad/datalad/cmdline/main.py", line 528, in main
    ret = cmdlineargs.func(cmdlineargs)
  File "/home/vanessa/Desktop/Code/datalad/datalad/interface/base.py", line 777, in call_from_parser
    ret = list(ret)
  File "/home/vanessa/Desktop/Code/datalad/datalad/interface/utils.py", line 413, in generator_func
    allkwargs):
  File "/home/vanessa/Desktop/Code/datalad/datalad/interface/utils.py", line 552, in _process_results
    for res in results:
  File "/home/vanessa/Desktop/Code/datalad/datalad/metadata/search.py", line 1264, in __call__
    for r in _search_from_virgin_install(dataset, query):
  File "/home/vanessa/Desktop/Code/datalad/datalad/metadata/search.py", line 234, in _search_from_virgin_install
    raise exc_info[1]
  File "/home/vanessa/Desktop/Code/datalad/datalad/metadata/search.py", line 1262, in __call__
    "this repository as a DataLad dataset" % ds.path)
datalad.support.exceptions.NoDatasetFound: This does not seem to be a dataset (no DataLad dataset ID found). 'datalad create --force /home/vanessa/Desktop/Code/datalad' can initialize this repository as a DataLad dataset

```
**WTF Output** :open_file_folder:
<details><summary>DataLad 0.10.3.1.dev3387 WTF (datalad, python, system, environment, configuration, location, extensions, metadata_extractors, dependencies, dataset)</summary>

# WTF
## datalad 
  - full_version: 0.10.3.1.dev3387-g59e1e-dirty
  - version: 0.10.3.1.dev3387
## dataset 
  - id: None
  - metadata: <SENSITIVE, report disabled by configuration>
  - path: /home/vanessa/Desktop/Code/datalad
  - repo: GitRepo
## dependencies 
  - appdirs: 1.4.3
  - boto: 2.49.0
  - cmd:annex: 7.20190819+git2-g908476a9b-1~ndall+1
  - cmd:bundled-git: 2.20.1
  - cmd:git: 2.20.1
  - cmd:system-git: 2.23.0
  - cmd:system-ssh: 7.6p1
  - git: 3.1.1
  - gitdb: 4.0.2
  - humanize: 2.4.0
  - iso8601: 0.1.12
  - keyring: 21.2.1
  - keyrings.alt: 3.4.0
  - msgpack: 1.0.0
  - requests: 2.23.0
  - tqdm: 4.46.0
  - wrapt: 1.12.1
## environment 
  - LANG: en_US.UTF-8
  - PATH: /home/vanessa/anaconda3/bin:/home/vanessa/anaconda3/condabin:/home/vanessa/.rbenv/plugins/ruby-build/bin:/home/vanessa/.rbenv/shims:/home/vanessa/.rbenv/bin:/home/vanessa/anaconda3/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/usr/local/go/bin
## location 
  - path: /home/vanessa/Desktop/Code/datalad
  - type: dataset
## metadata_extractors 
  - annex: 
    - load_error: None
    - module: datalad.metadata.extractors.annex
    - version: None
  - audio: 
    - load_error: No module named 'mutagen' [audio.py:<module>:17]
    - module: datalad.metadata.extractors.audio
  - datacite: 
    - load_error: None
    - module: datalad.metadata.extractors.datacite
    - version: None
  - datalad_core: 
    - load_error: None
    - module: datalad.metadata.extractors.datalad_core
    - version: None
  - datalad_rfc822: 
    - load_error: None
    - module: datalad.metadata.extractors.datalad_rfc822
    - version: None
  - exif: 
    - load_error: No module named 'exifread' [exif.py:<module>:16]
    - module: datalad.metadata.extractors.exif
  - frictionless_datapackage: 
    - load_error: None
    - module: datalad.metadata.extractors.frictionless_datapackage
    - version: None
  - image: 
    - load_error: None
    - module: datalad.metadata.extractors.image
    - version: None
  - xmp: 
    - load_error: No module named 'libxmp' [xmp.py:<module>:20]
    - module: datalad.metadata.extractors.xmp
## python 
  - implementation: CPython
  - version: 3.7.4
## system 
  - distribution: debian/buster/sid
  - encoding: 
    - default: utf-8
    - filesystem: utf-8
    - locale.prefered: UTF-8
  - max_path_length: 290
  - name: Linux
  - release: 5.3.0-51-generic
  - type: posix
  - version: #44~18.04.2-Ubuntu SMP Thu Apr 23 14:27:18 UTC 2020
</details>
        

<details>

## Environment
 - **LANG**: en_US.UTF-8
 - **DISPLAY**: :0
 - **USER**: vanessa
 - **PWD**: /home/vanessa/Desktop/Code/datalad
 - **HOME**: /home/vanessa
 - **TERM**: xterm-256color
 - **SHELL**: /bin/bash
 - **PATH**: /home/vanessa/anaconda3/bin:/home/vanessa/anaconda3/condabin:/home/vanessa/.rbenv/plugins/ruby-build/bin:/home/vanessa/.rbenv/shims:/home/vanessa/.rbenv/bin:/home/vanessa/anaconda3/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/usr/local/go/bin

## System
 ```python
{'python': {'version': '3.7.4', 'implementation': 'CPython'}, 'system': {'type': 'posix', 'name': 'Linux', 'release': '5.3.0-51-generic', 'version': '#44~18.04.2-Ubuntu SMP Thu Apr 23 14:27:18 UTC 2020', 'distribution': 'debian/buster/sid', 'encoding': OrderedDict([('default', 'utf-8'), ('filesystem', 'utf-8'), ('locale.prefered', 'UTF-8')])}}
```
</details>

generated by [HelpMe](https://vsoch.github.io/helpme/)
HelpMe Github Issue: md5.21bd5a3afdec1e16b4ab492408d83d78

**Comment by yarikoptic** [comment]
> We would do the check for the issue existing (I think) best with a GitHub action (not really possible on the command line, esp. without a GitHub token). 

GitHub action could do indexing of the fingerprints and pushing them back into repo with associated issues/titles. helpme client could query/cache the index locally to match local fingerprint before submitting to suggest a list of possible matches

**Comment by yarikoptic** [comment]
But all such fingerprinting could end up being a project/effort by itself ;-)

**Comment by vsoch** [comment]
Yeah I think I remember I implemented this already? See https://github.com/rseng/github-support.
